Use npm device.js instead of git device.js

Change `package.json` to use `    "device.js": "^2.3.0",` instead.

The git reference now pulls something without a device.js to import.

Thanks very much, this helped solve the following error. It was pretty difficult to find a solution via google so hopefully this comment will lead people here :

`ERROR in ./webpack/dev_helpers/CordovaDeviceRouter.js
Module not found: Error: Can't resolve 'device.js' in '/.../webpack/dev_helpers'
 @ ./webpack/dev_helpers/CordovaDeviceRouter.js 1:17-37
 @ multi ./src/main.js ./webpack/dev_helpers/CordovaDeviceRouter.js`

Additionally, you will want to edit CordovaDeviceRouter.js from the following format
**original :** deviceJs.desktop()
**correct :** deviceJs.device.desktop

the end of the file should look like this :
`if (deviceJs.device.desktop)
	bodyPrepend("browser")
else if (deviceJs.device.android)
	bodyPrepend("android")
else if (deviceJs.device.ios)
	bodyPrepend("ios")
else
	alert("Platform not found!")`

--- a/template_src/package.json
+++ b/template_src/package.json
@@ -14,7 +14,7 @@
     "chokidar": "^1.6.1",
     "clean-webpack-plugin": "^0.1.15",
     "css-loader": "^0.28.4",
-    "device.js": "git+https://github.com/matthewhudson/device.js.git",
+    "current-device": "^0.6.0",
     "epipebomb": "^1.0.0",
     "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.11.2",

--- a/template_src/webpack/dev_helpers/CordovaDeviceRouter.js
+++ b/template_src/webpack/dev_helpers/CordovaDeviceRouter.js
@@ -1,4 +1,4 @@
-const deviceJs = require("device.js")
+const deviceJs = require('current-device').default
 
 function bodyPrepend(platform) {
 	let body = document.querySelector("body"),

